docs/3.0/actions.md

```diff
@@ -103,6 +103,49 @@ def handle(query:, fields:, current_user:, resource:, **args)
 end
 ```
 
+## Passing Params to the Action Show Page
+When working with resource index or show views, it's often necessary to pass parameters to an action.
+
+One particular example is when you'd like to populate a field in that action with some particular value based on that param.
+
+```ruby
+class Action
+  def fields
+    field :some_field, as: :hidden, default: -> { if previous_param == yes ? :yes : :no}
+  end
+end
+```
+Example
+
+Consider the following scenario:
+
+1. Navigate to http://localhost:3030/admin/resources/users?hey=ya.
+2. Attempt to run the dummy action.
+You should be able to access the `hey` parameter and retrieve the value `ya`.
+
+#### Implementation
+
+To achieve this, we'll utilize the `request.referer` object and extract parameters from the URL. Here is how to do it:
+
+```ruby
+class Action
+  def fields
+    field :some_field, as: :hidden, default: -> {
+      parent_params = URI.parse(request.referer).query.split("&").map { |param| param.split("=")}.to_h.with_indifferent_access
+
+      if parent_params[:hey] == 'ya'
```

Comment:
Let's add these as comments above each code line that we'll have in the code example.

Reply:
Ok